index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>OAT | Portfolio</title>
+  <link rel="shortcut icon" href="./images/icons/fav.png" type="image/x-icon">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+      <a href="index.html">Oluwatoyin Olaoye</a>
+
+      <nav class="nav-link">
+        <ul class="link">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+        <a href="mailto:mindrepopps@gmail.com"><img src="./images/icons/mail_box.svg" alt="email destination"></a>
+      </nav>
+
+      <img src="./images/icons/menu_icon.png" alt="mobile menu bar">   
```

Comment:
Please move the hamburger menu inside the `<nav>` tag above.

<img width="379" alt="Screen Shot 2022-08-18 at 8 18 55" src="https://user-images.githubusercontent.com/82426682/185404786-3745b250-ebb1-46b5-8a5a-6f388e60e0c6.png">